### ISSUE TITLE
Acc of the validation set and training set only 0.0025 in the initial stage when fine-tuning the pre-trained model

### ISSUE THREAD

**Issue by PhyllisJi** [espnet/espnet#3356]
I fine-tune the pre-trained ASR models in the espnet1.
I use this command：
 ${cuda_cmd} --gpu ${ngpu} ${expdir}/train.log \
        asr_train.py \
        --ngpu ${ngpu} \
        --preprocess-conf ${preprocess_config} \
        --config ${train_config} \
        --backend ${backend} \
        --outdir ${expdir}/results \
        --tensorboard-dir tensorboard/${expname} \
        --debugmode ${debugmode} \
        --dict ${dict} \
        --debugdir ${expdir} \
        --minibatches ${N} \
        --verbose ${verbose} \
        --resume ${resume} \
        --train-json ${feat_tr_dir}/data_${bpemode}${nbpe}.json \
        --valid-json ${feat_dt_dir}/data_${bpemode}${nbpe}.json \
        **--enc-init ${recog_model} \
        --enc-init-mods 'encoder.' \
        --dec-init ${recog_model} \
        --dec-init-mods 'decoder.'**

May I ask what went wrong? From the perspective of acc, it does not use the parameters of the pre-trained model, otherwise the acc cannot almost start from 0.

**Comment by sw005320** [comment]
Did you use the same vocabulary?
If not, it would happen.

Also, easy debugging is to set the learning rate zero.
If the same thing still happens, then some part would be initialized unintentionally.
